### PR TITLE
[ONNX] Fix for expand -1 dim value

### PR DESCRIPTION
**Pull request by neginraoof**
PyTorch expand allows size with -1 dim value. -1 dim value means to infer the dimension from input tensor. This can be exported to ONNX expand with 1 dim value since ONNX expand supports two-way broadcast.

### CODE REVIEW

**Comment by neginraoof** [comment]
cc @abgoswam. This fixes the Bert-NER model export

**Comment by neginraoof** [comment]
Failure at: test/onnx/test_pytorch_onnx_caffe2.py::TestCaffe2BackendEmbed_opset9::test_index_copy
I don't think this is related to the PR updates.

**Review by abgoswam — APPROVED** [review]
:shipit:

**Comment by houseroad** [comment]
test is failing

**Comment by BowenBao** [comment]
Looks like this is failing the test_index_copy, where expand is used to export index_copy.

File: torch/onnx/symbolic_opset9.py [hunk 1]
```diff
@@ -305,6 +305,15 @@ def expand(g, self, size, implicit):
     size = sym_help._maybe_get_const(size, 'is')
     if not sym_help._is_value(size):
         size = g.op("Constant", value_t=torch.LongTensor(size))
+    else:
+        # Expand with -1 dim value means dim is unchanged.
+        # Since onnx::expand supports two-way broadcasting,
+        # -1 dim value can be exported to onnx as 1
+        list_ = stack(g, size, 0)
```
Comment (BowenBao) [inline torch/onnx/symbolic_opset9.py]: Need to check if `size` is list before applying stack. it seems `size` could be a tensor.

**Comment by houseroad** [comment]
Mar 08 10:01:45 test/onnx/test_operators.py::TestOperators::test_expand FAILED           [  2%]

**Comment by neginraoof** [comment]
cc @houseroad 
Can we merge this please? Thanks

**Review by houseroad — APPROVED** [review]
Looks good.

**Pull request closed without merging**